samples/community/samuelIkoli/template.json

```diff
@@ -0,0 +1,34 @@
+{
+  "pageDimensions": [
+    300,
+    400
+  ],
+  "bubbleDimensions": [
+    10,
+    10
+  ],
+  "customLabels": {},
+  "fieldBlocks": {
+    "MCQBlock1": {
+      "fieldType": "QTYPE_MCQ4",
+      "origin": [
+        58,
+        296
+      ],
+      "fieldLabels": [
+        "q1",
+        "q2",
+        "q3",
+        "q4",
+        "q5",
+        "q6",
+        "q7",
+        "q8",
+        "q9",
+        "q10"
```

Comment:
looks like the actual OMR image has more questions available. Can you configure the complete template for all questions visible? Also if possible add one more image(of same layout) to test your template with.
 
Also you can use the short syntax "q1..10", etc for consecutive questions.